code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -26,10 +26,15 @@
 	var/datum/action/innate/slime_pick_up/slime_up_action = new
 	var/datum/action/innate/feed_slime/feed_slime_action = new
 	var/datum/action/innate/monkey_recycle/monkey_recycle_action = new
+	var/datum/action/innate/feed_potion/potion_action = new
+	var/datum/action/innate/hotkey_help/hotkey_help = new
+	var/datum/action/innate/slime_scan/scan_action = new
```

Comment:
These still need re-arranged in the same manner as TG has them. 

When we do ports, we try to have code 1:1. It makes it a lot easier to gauge/judge. The only changes we make are because of (1) mistakes (2) Balance changes (3) other reasons that are explicitly stated.

Reply:
sorry I was going to do but after I changed the names I forgot to switch them.